package.json

```diff
@@ -44,6 +44,10 @@
     "firebase-deploy:staging": "firebase deploy --project=staging --only functions:CI,firestore:rules",
     "firebase-deploy:master": "firebase deploy --project=production --config=firebase-production.json --only functions:CI,firestore:rules",
     "firebase-deploy-backup-function": "firebase deploy --project=production --only functions:scheduledFirestoreBackup",
+    "firebase-deploy-deepdao-function:default": "firebase deploy --project=default --only functions:scheduledDeepDaoOrganizationListUpdate",
+    "firebase-deploy-deepdao-function:development": "firebase deploy --project=development --only functions:scheduledDeepDaoOrganizationListUpdate",
+    "firebase-deploy-deepdao-function:staging": "firebase deploy --project=staging --only functions:scheduledDeepDaoOrganizationListUpdate",
+    "firebase-deploy-deepdao-function:production": "firebase deploy --project=production --only functions:scheduledDeepDaoOrganizationListUpdate --config=firebase-production.json",
```

Comment:
To keep this on the radar

![image](https://user-images.githubusercontent.com/30693990/185588023-6b838a01-5cb5-4279-bd07-378c916f9f86.png)

https://github.com/PrimeDAO/prime-deals-dapp/pull/1135/files#r944096086